indico/modules/categories/controllers/management.py

```diff
@@ -149,6 +149,12 @@ def _get_defaults(self):
         return FormDefaults(self.category, acl=acl, managers=managers, event_creators=event_creators)
 
 
+class RHCategoryMoveContents(RHManageCategoryBase):
+    def _process(self):
+        return WPCategoryManagement.render_template('management/move_category_contents.html', self.category,
+                                                    active_menu_item='')
```

Comment:
this shouldn't be a kwarg and if this thing is always shown inside a dialog you probably don't need a WP at all but can use `jsonify_template()` instead.